apps/src/templates/curriculumCatalog/ExpandedCurriculumCatalogCard.jsx

```diff
@@ -136,7 +136,7 @@ const ExpandedCurriculumCatalogCard = ({
                         {professionalLearningProgram && (
                           <TextLink
                             text={i18n.facilitatorLedWorkshops()}
-                            href={professionalLearningProgram}
+                            href={`https://${professionalLearningProgram}`}
```

Comment:
Would it be possible to update the URL constants instead? I'm wondering if it might get confusing if this is handled differently than down below with `href={selfPacedPlCourseOfferingPath}`?

Reply:
Would this mean that the professionalLearningProgram would have to be inputted again through levelbuilder to account for the change?

Reply:
Oh shoot, I'm not sure. @davidsbailey Could I get your thoughts on this please?

Reply:
Turner, your idea sounds best to me. it should be possible to run a command from rails console on your local machine to update all of these fields at once, and serialize the changes to the filesystem.
```ruby
CourseOffering.each do |co|
  co.professional_learning_program = ...
  co.write_serialization
end
```
then you can send the result for code review.

Reply:
I'll make this change! Thank you both!